sdk/metricsadvisor/tests.yml

```diff
@@ -0,0 +1,20 @@
+trigger: none
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: metricsadvisor
+    EnvVars:
+      METRICSADVISOR_ACCOUNT_NAME: js-metrics-advisor
```

Comment:
Curious about this - does this hard-code a dependency on a single resource?  Are we using ARM templates for creating resources?

Reply:
We are using a pre-existing resource in MA tests (not only in .NET, but in other languages as well). This is necessary because the service takes a lot of time to create the resource (30 min+), and some extra time to ingest data, detect anomalies, and trigger alerts. Since we have a limited amount of time when running tests, we decided it would be better to use a pre-existing resource.